packages/sampler/.storybook/preview.js

```diff
@@ -0,0 +1,22 @@
+import {addDecorator} from '@storybook/react';
+import {configureActions} from '@enact/storybook-utils/addons/actions';
+import {withKnobs} from '@enact/storybook-utils/addons/knobs';
+import {DocsPage, DocsContainer} from '@enact/storybook-utils/addons/docs';
+import { themes } from '@storybook/theming';
+
+import Environment from '../src/Environment';
+
+configureActions();
+addDecorator(withKnobs);
+export const parameters = {
+	knobs: {
+		timestamps: true,
+	},
+	docs: {
+		container: DocsContainer,
+		page: DocsPage,
+		iframeHeight: 300,
+		theme: themes.light
```

Comment:
Where these settings come from?

Reply:
Those are found in storybook examples, 
https://github.com/storybookjs/storybook/blob/next/examples/official-storybook/preview.js#L161
https://github.com/storybookjs/storybook/blob/next/examples/vue-cli/.storybook/preview.js#L3